### DISCUSSION TITLE
Remote Joined columns are not showing in where condition

### DISCUSSION THREAD

**Discussion by santhoshkumarch** [hasura/graphql-engine#8863] [Question]
I have added two PostgreSQL DB in a DB manager, as shown in the image,
<img width="141" alt="Screenshot 2022-08-09 at 11 27 56 AM" src="https://user-images.githubusercontent.com/21010099/183575346-55c0b088-c639-40c4-ad6c-b49e3d52f9a8.png">
 
and i have make a join from the primary db table  to secondary db table  - the columns are showing but i cant able to do `where` condition by using the secondary db table,
<img width="153" alt="Screenshot 2022-08-09 at 11 30 13 AM" src="https://user-images.githubusercontent.com/21010099/183575659-fd4e5b69-d71e-4a78-96c4-d21ead763d55.png">

<img width="125" alt="Screenshot 2022-08-09 at 11 30 45 AM" src="https://user-images.githubusercontent.com/21010099/183575719-9ff6fc91-9b0d-4611-a03e-ac6ae364075d.png">

<img width="674" alt="Screenshot 2022-08-09 at 11 36 55 AM" src="https://user-images.githubusercontent.com/21010099/183576705-afc26c72-0471-4204-b5f0-b69ab3aae7bc.png">

<img width="929" alt="Screenshot 2022-08-09 at 11 37 05 AM" src="https://user-images.githubusercontent.com/21010099/183576651-01b6a4e8-72d6-451d-b78e-5cf037a9d9f4.png">

Anyone pls suggest me a solution for this

**Comment by santhoshkumarch** [comment]
Any update on this?

**Comment by robertjdominguez** [comment]
@santhoshkumarch, I'm converting this to a discussion and adding a couple of labels 👍 

Can you check [this page in our docs](https://hasura.io/docs/latest/schema/postgres/remote-relationships/remote-source-relationships/) and see if it helps you resolve your question?

**Reply by santhoshkumarch** [reply]
that document shows how to create remote db relationship only, my question is `i cant able to see the connected table in where condition` - can you pls check @robertjdominguez 

Thanks
